webflux-oauth2login/src/main/resources/application.properties

```diff
@@ -1,3 +1,2 @@
-okta.oauth2.issuer=https://dev-820159.okta.com/oauth2/default
-okta.oauth2.clientId=0oa1ec63zpvcYtLGj357
-okta.oauth2.clientSecret=qwmW0BXLlqWwPdrwOKHNigNGbM21whD3wjOaOucU
```

Comment:
@moksamedia Why did you remove the client secret? Was that intentional? 

I know Spring Security 5.2 added support for PKCE, which makes it possible to do auth code flow without a secret. However, it doesn't work with our starter when I [last tried it](https://github.com/okta/okta-spring-boot/issues/132).

Reply:
It seems to be working for me without it. That's why I removed it. I just double-checked it. Can you try it to verify?

Reply:
If I remove it and try locally, I get a 401 in the browser instead of a redirect. I'll add it back in.

<img width="1220" alt="Screen Shot 2019-11-22 at 11 56 14 AM" src="https://user-images.githubusercontent.com/17892/69452561-373b4580-0d1f-11ea-9e08-1330b57c1499.png">

Reply:
I think the tutorial just uses HTTPie with a token. I wasn't really writing to support client login.

Reply:
Ooops. Sorry. I tested it as a resource server, not as OAuth2 Login. Written too many of the resource server tutorials.